src/release/release-notes.md

```diff
@@ -1,21 +1,158 @@
+Did not use "Stabilized APIs" from Tracking issue for release notes of #67521: Tracking issue for const `alloc::Layout` <https://github.com/rust-lang/rust/issues/134725>
+Did not use "Category (e.g. Language, Compiler, Libraries, Compatibility notes, ...)" from Tracking issue for release notes of #132187: Add Extend impls for tuples of arity 1 through 12 <https://github.com/rust-lang/rust/issues/133975>
+```
 
+These lines typically mean that someone either hasn't updated or used a keyword not in the list above, for example:
+
+* Libraries should be Library
+* Stabilized APIs is not yet supported (FIXME!)
+* Category ... needs updating.
+
+The runner of the tool should go update these issues with the right contents
+(e.g., renaming/categorizing). The **source of truth** is the issue, not the
+local output of the tool -- this is important as we want to automate tool
+execution in the long run, so try to minimize the amount of local edits done on
+the resulting `relnotes.md`.
+
+The tool's standard output is directed to `relnotes.md` above, which will
+currently contain formatted headers for each of the sections with content
+pulled from the "Tracking issue for ..." **or** standard content (PR/issue
+title + link) if there is no "tracking issue for ..." found by the tool.
+
+If an issue is missing, see Step 3 (tag it and re-run the tool). If an issue is
+present *but shouldn't be*, the best thing is to tag it with relnotes (or find
+the pre-existing relnotes tracking issue) and *close that tracking issue*. This
+will also drop the item from the tool's output.
+
+## Release team: Step 5: Publish relnotes PR
+
+See example from 1.84: https://github.com/rust-lang/rust/pull/134568
+
+Take the `relnotes.md` you have locally (typically without library
+stabilizations in today's world, you'll add them at a later point -- we want
+the copy without them as early as possible), and insert it at the top of
+RELEASES.md in rust-lang/rust, and open a new PR with those contents. You can
+`r?` the owner for actually publishing the release for this cycle and cc the
+release team.
+
+Include a link to this document (https://forge.rust-lang.org/release/release-notes.html)
+in the PR description, pointing at step 6 (i.e., prefer suggesting updates not on the PR).
+
+The next release team meeting should also discuss this PR for selecting blog
+post topics (see below for blog post process).
+
+## All: Step 6: Incorporate edits from relnotes PR
+
+You'll typically get a lot (several dozen) of comments on the PR with typo
+fixes, suggestions for alternative text, etc. A good strategy is to try to
+update the originating issues for issues/PRs (or file them and update them),
+essentially matching the iteration already done locally in step 4. The longer
+state stays in issues the easier it is to notice and incorporate updates from
+those into the PR (just rerun the tool).
```

Comment:
FWIW I haven't seen this happen so far, is that new?

Reply:
"new" maybe a strong descriptor -- it's not the typical process, for sure. Last I did relnotes (1.82?) I did do this for at least a few rounds of feedback.